tee-worker/cli/demo_sidechain.sh

```diff
@@ -129,7 +120,7 @@ echo ""
 
 # Send funds from Alice to Bobs account, on worker 2.
 echo "* Second transfer: Send ${AMOUNTTRANSFER} funds from Alice's incognito account to Bob's incognito account (on worker 2)"
-$CLIENTWORKER2 trusted --mrenclave ${MRENCLAVE} --direct transfer ${ICGACCOUNTALICE} ${ICGACCOUNTBOB} ${AMOUNTTRANSFER}
+$ClientWorker1 trusted --mrenclave ${MRENCLAVE} --direct transfer ${ICGACCOUNTALICE} ${ICGACCOUNTBOB} ${AMOUNTTRANSFER}
 echo ""
```

Comment:
Mentions of "worker 2" in output and comments are now a lie; please amend those as well.

Reply:
Reverted changes, No need to update